Lib/test/test_gettext.py

```diff
@@ -727,6 +751,169 @@ def test_unicode_msgstr_with_context(self):
         eq(t, "Hay %s ficheros (context)")
 
 
+class TemplateStringsTestsMixin:
+    def setup_tstrings_test(self, *, expect_translations=False):
+        # We use this weird naming of the gettext functions here to allow
+        # easy extraction of the .po file using pygettext; see the comment
+        # next to the po file content near the bottom of this file on how
+        # to regenerate it.
+        self.gettexT = self.t.gettext
+        self.ngettexT = self.t.ngettext
+        self.pgettexT = self.t.pgettext
+        self.npgettexT = self.t.npgettext
```

Comment:
I don't expect to keep these weird names, but for the sake of making it easier to update the tests in the future, I'd probably go for something like `t_gettext` etc. in here.